.env.docker.example

```
@@ -0,0 +1,26 @@
+FRONTEND_URL=http://localhost
+BACKEND_URL=http://localhost
+CORS_ORIGINS=http://localhost
+
+COOKIE_SECURE=false
+COOKIE_SAMESITE=lax
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The COOKIE_SAMESITE key should go before the COOKIE_SECURE key

(UnorderedKey)